elf_reader.go

```diff
@@ -207,8 +207,8 @@ func (ec *elfCode) loadPrograms(progSections map[elf.SectionIndex]*elf.Section,
 
 		if btfSpec != nil {
 			spec.BTF, err = btfSpec.Program(sec.Name, length)
-			if err != nil && !xerrors.Is(err, btf.ErrNoExtendedInfo) {
-				return nil, xerrors.Errorf("program %s: %w", sec.Name, funcSym.Name, err)
+			if err != nil && !errors.Is(err, btf.ErrNoExtendedInfo) {
+				return nil, fmt.Errorf("program %s %s: %w", sec.Name, funcSym.Name, err)
```

Comment:
Oops! I think we should drop sec.Name instead:

```suggestion
				return nil, fmt.Errorf("program %s: %w", funcSym.Name, err)
```